python/tvm/relay/qnn/op/qnn.py

```diff
@@ -53,6 +54,9 @@ def requantize(data,
     output_zero_point: tvm.relay.Expr
         The zero point of the output tensor.
 
+    axis : int
+        The channel axis for quantization.
```

Comment:
Can we add a comment on what a -1 (default) argument value means for `axis`?

Reply:
Good catch! This just means the last axis (similar to other ops). I will add.

Reply:
Gotcha, that makes sense! Just want to avoid confusion with a "special case value".